torchci/scripts/check_alerts.py

```diff
@@ -268,7 +268,7 @@ def clear_alerts(alerts: List[Any]) -> bool:
 
 # We need to clear alerts is there is a commit that's all green is before a commit that has a red
 # If there's pending things after the all green commit, that's fine, as long as it's all green/pending
-def should_clear_alerts(sha_grid: Any):
+def master_is_green(sha_grid: Any):
```

Comment:
```suggestion
def trunk_is_green(sha_grid: Any):
```

We will probably move to main as default branch at some point, so let's give ourselves less work to do later

Reply:
let's do it